utils/image_saver.py

```diff
@@ -38,8 +39,13 @@ def save_image() -> None:
         ) if answer == 'TID Repo' else input("Please enter download link: ")
 
         path: str = os.path.join(os.getcwd(), "data", "images")
+    
+        if not os.path.exists(path):
```

Comment:
This partially solve the problem for `save` command but the bug is still there for the `download` command. You can put this if statement in `downloader.py`. Should solve the problem. 

Put it somewhere around *here*.

![image](https://user-images.githubusercontent.com/44261822/126858862-763eabba-18f8-41c5-b33d-f955f5d1f167.png)